src/Shouldly/ShouldlyConfiguration.cs

```diff
@@ -44,20 +45,20 @@ static ShouldlyConfiguration()
         /// </summary>
         public static IDisposable DisableSourceInErrors()
         {
-            CallContext.LogicalSetData("ShouldlyDisableSourceInErrors", true);
+            ShouldlyDisableSourceInErrors.Value = true;
             return new EnableSourceInErrorsDisposable();
         }
 
         public static bool IsSourceDisabledInErrors()
         {
-            return (bool?) CallContext.LogicalGetData("ShouldlyDisableSourceInErrors") == true;
+            return ShouldlyDisableSourceInErrors.Value == true;
```

Comment:
What's the plan with `ShouldlyDisableSourceInErrors`? I can't see it anywhere in your branch? Is this part WIP? I noticed [this article](http://www.cazzulino.com/callcontext-netstandard-netcore.html) that can achieve a polyfill for `CallContext` on non-Windows platforms.